Vagrantfile

```diff
@@ -74,43 +27,20 @@ Vagrant.configure(2) do |config|
     config.vm.define "centos" do |centos|
         centos.vm.box = "centos/atomic-host"
         centos.vm.hostname = "centosah-dev"
-        centos.vm.provision "shell", inline: $centos
```

Comment:
I really don't use vagrant so this may be totally wrong, but do we need something to provision the correct ostree repo/ref onto the vagrant box?  The deploy head-1 was removed but it seems like we still need something to deploy HEAD pointing to the correct ref

Reply:
Hmm...that Vagrant box should always be up to date, but if you already have the box on your host, you are going to need an extra set of provision steps to get it current.

The same can be said for any of the boxes, actually.

Good catch!  Back to the drawing board...